CMakeLists.txt

```diff
@@ -24,11 +25,13 @@ cmake_minimum_required(VERSION 3.1)
 
 project(treesheets)
 
+include(GNUInstallDirs)
+
+set(TREESHEETS_RELOCATABLE_INSTALLATION OFF CACHE BOOL "Enable to look for data relative to the path treesheets was run from, instead of hardcoding paths from CMAKE_INSTALL_PREFIX")
```

Comment:
I wonder if we need this option. The reason I introduced it is that I wanted to prevent relocatable builds from loading the locales from install prefix (e.g. when user has an older version installed on their system and fresh one installed into a testing directory).

Reply:
Now the option just serves to toggle between the FHS-compatible layout and the layout used on Apple and Windows.